lib/beaker-pe/install/pe_utils.rb

```diff
@@ -92,11 +92,11 @@ def install_ca_cert_on(host, opts)
             @cert_cache_dir ||= Dir.mktmpdir("master_ca_cert")
             local_cert_copy = "#{@cert_cache_dir}/ca.pem"
             step "Copying master ca.pem to agent for secure frictionless install" do
-              ca_pem_dir = '/etc/puppetlabs/puppet/ssl/certs'
-              ca_pem_path = "#{ca_pem_dir}/ca.pem"
-              scp_from(master, ca_pem_path , @cert_cache_dir) unless File.exist?(local_cert_copy)
+              agent_ca_pem_dir = "#{host['puppetpath']}/ssl/certs"
+              master_ca_pem_path = "/etc/puppetlabs/puppet/ssl/certs/ca.pem"
+              scp_from(master, master_ca_pem_path , @cert_cache_dir) unless File.exist?(local_cert_copy)
               on(host, "mkdir -p #{ca_pem_dir}")
```

Comment:
agent_ca_pem_dir now I think

Also looks like there are some spec failures.  One of them was this ^; the other I think was matching excapse changes.